x-pack/plugin/sql/qa/server/src/main/resources/datetime.csv-spec

```diff
@@ -1570,3 +1570,173 @@ SELECT CAST (CAST (birth_date AS VARCHAR) AS TIMESTAMP) a FROM test_emp WHERE YE
 ---------------
 1965-01-03T00:00:00Z
 ;
+
+
+
+// datetime ranges
```

Comment:
All the tests here are only using `AND`. I'd assume date math would also work with `OR` and `NOT`?

Reply:
Good point, adding them